pkg/netns/netns.go

```diff
@@ -23,7 +22,7 @@ func RemoveIfFromNetNSIfExists(netNS ns.NetNS, ifName string) error {
 	return netNS.Do(func(_ ns.NetNS) error {
 		l, err := netlink.LinkByName(ifName)
 		if err != nil {
-			if errors.As(err, &netlink.LinkNotFoundError{}) {
+			if _, ok := err.(netlink.LinkNotFoundError); ok {
```

Comment:
why did you change the `errors.As`? Was it not working as intended?

Reply:
ah, I don't remember it, but you're right. updated now.